cli/compile.vsh

```diff
@@ -30,8 +30,23 @@ if additional_args.len > 0 {
 	exit(1)
 }
 
-// Change to the hero directory
-hero_dir := os.join_path(os.home_dir(), 'code/github/incubaid/herolib/cli')
+// Determine the hero directory dynamically
```

Comment:
why did we introduce this?
should not be needed
it means people checked out in places where they shouldn't
but anyhow its ok

Reply:
Since Sabrina moved the repository from `freeflowuniverse` to `incubaid`, the `cli/compile.vsh` script now raises an error due to the incorrect path.
We have two possible solutions:

1. **Update all imports and symlinks** to use `incubaid` instead of `freeflowuniverse`.
2. **Make the script more dynamic** so it can detect the correct location on the user’s machine and work with both paths.